README.md

```diff
@@ -131,184 +149,23 @@ darwin-rebuild switch
-  be expected to be up or reliable, because chances are someone's working on
-  it.
-
-#### IMPORTANT
-
-If you want to avoid hours of building, it's recommended that you set up our binary cache.
-Instructions for this are on the [cachix page](https://disciplina.cachix.org).
-
-You need a patched version of nix. Upstream PR is currently pending. The correct
-version is provided as part of the overlay used in this repo. To install it,
-`cd` into repo base, and:
-
-```sh
-nix-env -f pkgs.nix -iA nix
-```
-
-#### Creating the deployment
-
-To create a new testing deployment, run the following:
-
-```sh
-nixops create deployments/cluster.nix -d disciplina
-nixops set-args --argstr domain yourname.dscp.serokell.review -d disciplina
-nixops deploy -d disciplina
-```
-
-Where `-d disciplina` can be replaced by any name you like, such as
-`discplina-testing` or `disciplina-kawaii`.
-
-To push changes to the cluster, re-run the last command.
-
-When not actively working on your cluster, you should shut it down:
-
-```sh
-nixops stop -d disciplina --confirm
-```
-
-You can start it back up with:
-
-```sh
-nixops start -d disciplina --confirm
-```
-
-To clean up your cluster, which you should do when you're done with it:
-
-```sh
-nixops destroy -d disciplina --confirm
-```
-
-#### Deploying locally
 
-You should only deploy to AWS if you have a good reason for it. You can deploy
-to your local computer by spawning VirtualBox machines, which is much faster.
 
-The entire cluster will need ~2Gb of RAM and 13Gb of HDD space once deployed and running.
-
-##### Setup
-
-You will need to install VirtualBox.
-
-On NixOS, add the following to your `configuration.nix`:
-```
-  virtualisation.virtualbox.host.enable = true;
-```
-
-For other Linux distributions, it's probably in your repositories.
-
-On other systems, visit the [VirtualBox
-website](https://www.virtualbox.org/wiki/Downloads) to download and install.
-
-##### Important!
-
-From the nixops manual:
-> Note that for this to work the vboxnet0 network has to exist - you can add it in
-> the VirtualBox general settings under Networks - Host-only Networks if
-> necessary.
-
-##### Deployment
-
-Replace `<name>` with what you want to call this deployment. For example,
-`disciplina-vm` or `disciplina-local`. Use a unique name.
-
-```sh
-nixops create deployments/cluster.nix -d <name>
-nixops set-args --argstr hostType virtualbox -d <name>
-```
-
-You can stop and start the cluster same as above. This will stop all VMs. You
-will also need to start the deployment after rebooting your machine.
-
-#### Deploying to production
-
-Do as above to create a deployment, skip the second step, and run this instead:
-
-```sh
-nixops set-args --argstr env production -d disciplina
-nix-shell --argstr env production --run 'nixops deploy -d cluster -s state/cluster.nixops'
-```
-
-Re-run last command to push new changes.
-
-### Overriding dependencies
```

Comment:
This section is still relevant.

Reply:
I think we have all of that encapsulated in `setdeps.sh`, and it is explained both in other parts of the readme and in the script itself, so I’m not sure there is any point in keeping this section.

Reply:
`setdeps.sh` doesn't cover all use cases. In particular, it doesn't cover any use case where you don't want to use the same branch/tag/location for every repo. I think this is a good description and we should have the relevant deployment step point here instead.